### PR TITLE
enh(UI): Add cancel icon into SearchField to clear filter easily by one click

### PR DESCRIPTION
**Pull request by Jordan91**
## Description

For now , we can't clear filter easily, we must refresh or another action to start new filter into the search field

**Fixes** # (issue)

Add cancel icon on the search field to clear easily filter

## Type of change

- [x] New functionality (non-breaking change)

## Target serie

- [x] 21.10.x (master)

<h2> How this pull request can be tested ? </h2>

Go on the search field bar into ressource status, set a custom filter then you can try to delete filter by clicking on the Cancel icon at the end of the search field bar

### CODE REVIEW

File: www/front_src/src/Resources/Filter/index.tsx [hunk 1]
```diff
@@ -22,13 +23,15 @@ import {
   Popper,
 } from '@material-ui/core';
 
+import IconButton from '@centreon/centreon-frontend/packages/centreon-ui/src/Button/';
 import { MemoizedFilter, SearchField } from '@centreon/ui';
```
Comment (Thebarda) [inline www/front_src/src/Resources/Filter/index.tsx:27]:
```suggestion
import { MemoizedFilter, SearchField, IconButton } from '@centreon/ui';
```

File: www/front_src/src/Resources/Filter/index.tsx [hunk 2]
```diff
@@ -48,6 +51,10 @@ const useStyles = makeStyles((theme) => ({
   autocompletePopper: {
     zIndex: theme.zIndex.tooltip,
   },
+  clearFilter: {
+    padding: 'revert',
+    position: 'inherit',
+  },
```
Comment (Thebarda) [inline www/front_src/src/Resources/Filter/index.tsx:57]: Why applying this style ? What happens if you remove this ?

**Review by Thebarda — COMMENTED** [review]
Do not forget to add translation to other messages.po

**Comment by Jordan91** [comment]
> Do not forget to add translation to other messages.po

Ok , no problem :)

File: www/front_src/src/Resources/Filter/index.tsx (hunk 2 above)
Comment (Jordan91) [reply www/front_src/src/Resources/Filter/index.tsx:56]: Without application style  they look like that: 
Capture d’écran 2021-09-16 à 15 43 16


With style clearFilter:
Capture d’écran 2021-09-16 à 15 41 38

In two case it's not perfect :/

Comment (Jordan91) [reply www/front_src/src/Resources/Filter/index.tsx:56]: > Without application style they look like that:
> Capture d’écran 2021-09-16 à 15 43 16
> 
> With style clearFilter:
> Capture d’écran 2021-09-16 à 15 41 38
> 
> In two case it's not perfect :/ but it's works , that's something ^^

Comment (Jordan91) [reply www/front_src/src/Resources/Filter/index.tsx:56]: BY closing PR on Frontend it's works as expected

**Comment by Jordan91** [comment]
Bad command line use to resolve conflicts

**Pull request closed without merging**